output/components.eslint.txt

```diff
@@ -48,6 +54,6 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/components/src/VirtualizedList/utils/tablerow.js
   41:3  warning  Unexpected console statement  no-console
 
-✖ 24 problems (21 errors, 3 warnings)
-  1 error, 0 warnings potentially fixable with the `--fix` option.
+✖ 26 problems (23 errors, 3 warnings)
+  2 errors, 0 warnings potentially fixable with the `--fix` option.
```

Comment:
need to fix that

Reply:
Fixed one, the other one is due to prettier, so we can't do anything. We should disable indentation linter rules to let prettier as the only source for indentation